xcode/Subconscious/Shared/Components/FirstRun/FirstRunCreateSphereView.swift

```diff
@@ -16,7 +16,7 @@ struct FirstRunCreateSphereView: View {
             VStack {
                 Spacer()
                 VStack(alignment: .center, spacing: AppTheme.padding * 2) {
-                    Text("This is your secret recovery phrase. You can use it to recover your data if you lose access.")
+                    Text("This is your secret recovery phrase. You can use it to recover your sphere if you lose access.")
```

Comment:
I moved back to "data" here because I changed up the design a bit. It's a lot of information _but_ I think this screen tells a much more coherent story now:

<img width="374" alt="Screenshot 2023-05-11 at 11 24 34 am" src="https://github.com/subconsciousnetwork/subconscious/assets/5009316/4306c686-49f0-4387-85e9-f2d9a202223e">